docs/EMERGENCY.md

```diff
@@ -0,0 +1,213 @@
+# Emergency Procedures for Yearn Finance
+
+## TLDR: [Emergency Check List](#emergency-checklist)
+
+## Introduction
+
+This document details the emergency procedures and guidelines that should take place during an emergency situation.
+
+The main goal of this document is to help minimize as much as possible a potential loss of funds in yearn platform and Smart Contracts during an emergency situation.
+
+## Definitions and Examples of Emergencies
+
+For the purposes of this document we define emergency situation to be:
+
+**_Any situation that may lead to a considerable amount of loss of funds for yearn users._**
+
+This is a non exhaustive list of possible emergency scenarios:
+
+1. Exploit in Vault/Strategy code that can cause a loss of funds for users
+2. Bug in Vault/Strategy code that may cause loss of funds for users
+3. An exploit in an underlying protocol that a yearn strategy is utilizing that may lead to loss of funds for the users
+4. A bug in an underlying protocol that a yearn strategy is utilizing that may lead to loss of funds for the users
+5. A key role like a strategist having the private keys stolen
+
+## Roles
+
+The following roles are identified and should be assigned during an emergency.
+
+- Facilitator
+- Strategist Lead
+- Multi-sig Operations
+- Web Lead
+- Core Dev Lead (Guardian)
+- Ops
+
+Some of these roles may be done by the same person up to two concurrent roles.
+
+### Facilitator:
+
+The person in this role will facilitate the emergency handling and ensure that the team follows the overall process and engages the correct teams and stakeholders to make the necessary quick decisions. This person can be anybody that is familiar with the process and is confident that he/she can drive the team to accomplish the steps needed. It's expected that this person has some experience either through real scenarios or drill training.
+
+### Multi-sig Operations:
+
+The person in this role will be tasked with ensuring that during the emergency scenario the different yearn teams multi-sig wallets can execute emergency transaction in a timely manner.
+
+Main responsibilities are:
+
+- Help clearing the queue on any pending operations once the War Room starts
+- Coordination of required signers so they can respond quickly to the queued transactions
+- Preparing or helping with transactions in the different multi-sigs
+  (dev.ychad.eth, brain.ychad.eth, ychad.eth)
+  Reference:
+  - [emergency-toolbox](https://github.com/yearn/emergency-toolbox)
+  - [CMO](https://github.com/yearn/chief-multisig-officer)
```

Comment:
https://github.com/poolpitako/strategists-ms can be added.
TBH, cmo and strategist-ms should move inside the yearn org.